### DISCUSSION TITLE
Can't redeem my tokens

### DISCUSSION THREAD

**Discussion by loicredon** [wormhole-foundation/wormhole#2308] [Problems / bugs] [answered]
I bridged some USDC from Polygon to Solana, but i can't redeem them

The hash: 0x6db71c626b424cfb99b8862e7447eb30f36a226631a3cd88301acb43accb508d

Thanks

**Comment by kev1n-peters** [comment, marked as answer]
These have already been redeemed: https://solana.fm/tx/hnLssEXXYr6CeCn5v9mHi3ESfiFZ87QMgaYLUUa1xNmA54GG7DELxA8mARptZ92U1q1dE24Yf7t6k3jPwSRTqot?cluster=mainnet-qn1
